bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/gems/CanopusFeature.java

```diff
@@ -4,70 +4,31 @@
 import edu.gemini.shared.util.immutable.Some;
 import edu.gemini.shared.util.immutable.None;
 import edu.gemini.spModel.gemini.gems.Canopus;
-import edu.gemini.spModel.guide.GuideProbe;
 import edu.gemini.spModel.obs.context.ObsContext;
 import edu.gemini.spModel.obscomp.SPInstObsComp;
-import edu.gemini.spModel.target.SPTarget;
 import edu.gemini.spModel.target.WatchablePos;
-import edu.gemini.spModel.target.env.GuideProbeTargets;
-import edu.gemini.spModel.target.env.TargetEnvironment;
 import edu.gemini.spModel.target.env.TargetEnvironmentDiff;
 import jsky.app.ot.tpe.*;
-import jsky.app.ot.tpe.feat.TpeGuidePosCreatableItem;
-import jsky.app.ot.util.BasicPropertyList;
-import jsky.app.ot.util.OtColor;
 import jsky.app.ot.util.PropertyWatcher;
 
-import javax.swing.*;
 import java.awt.*;
 import java.awt.geom.AffineTransform;
 import java.awt.geom.Area;
 import java.awt.geom.Point2D;
-import java.awt.geom.Rectangle2D;
-import java.awt.image.BufferedImage;
 import java.beans.PropertyChangeListener;
 import java.util.Collection;
 import java.util.Collections;
 
 /**
  * Draws the Canopus AO field of view and probe ranges.
  */
-public final class CanopusFeature extends TpeImageFeature implements PropertyWatcher, TpeModeSensitive, TpeDragSensitive {
+public final class CanopusFeature extends TpeImageFeature implements PropertyWatcher {
 
     private AffineTransform trans;
     private boolean isEmpty;
 
     // Color for AO WFS limit.
     private static final Color AO_FOV_COLOR = Color.RED;
-    private static final Color PROBE_RANGE_COLOR = OtColor.SALMON;
-
-    // Composite used for drawing items that block the view
-    private static final Composite BLOCKED = AlphaComposite.getInstance(AlphaComposite.SRC_OVER, 0.5F);
-
-    // Property used to control drawing of the probe ranges.
-    private static final BasicPropertyList props = new BasicPropertyList(CanopusFeature.class.getName());
-    private static final String PROP_SHOW_RANGES = "Show Probe Ranges";
-    static {
-        props.registerBooleanProperty(PROP_SHOW_RANGES, true);
-    }
-
-    private enum RangeDisplayMode {
```

Comment:
We no longer need a `RangeDisplayMode` due to elimination of CWFS1 and 2 ranges.